.github/workflows/release.yaml

```diff
@@ -6,8 +6,8 @@ on:
 env:
   go-version: 1.23
 jobs:
-  release:
-    name: release images
+  build:
```

Comment:
It looks better the job identifier remain "release" since its main purpose is release.

Reply:
@012xx 
How about this?